src/config/Config.zig

```diff
@@ -1076,6 +1076,15 @@ keybind: Keybinds = .{},
 /// This is currently only supported on macOS.
 @"window-title-font-family": ?[:0]const u8 = null,
 
+/// The text that will be displayed in the subtitle of the window. Valid values:
+///
+///   * `false` - Disable the subtitle.
+///   * `working-directory` - Set the subtitle to the working directory of the
+///      surface.
+///
+/// This feature is only supported on GTK with Adwaita enabled.
+@"window-subtitle": WindowSubtitle = .false,
```

Comment:
I made the default false. Easier for me to just enable than make a whole bunch of people disable :smile:

Reply:
I'm happy to reassess "zero config defaults" as more feedback arrives when something is shipped. I'm very supportive of the least impactful config being the default to start, and if we find many people are turning it on then we switch course.